crates/lib3h/src/engine/real_engine.rs

```diff
@@ -285,9 +394,19 @@ impl<'engine> RealEngine<'engine> {
                 // TODO #168
             }
             Lib3hClientProtocol::Connect(msg) => {
-                // Convert into TransportCommand & post to network gateway
-                let cmd = TransportCommand::Connect(msg.peer_uri, msg.request_id);
-                self.network_gateway.as_transport_mut().post(cmd)?;
+                // no-op ?
+                // send empty message so it connects
+                let cmd = GatewayRequestToChild::Transport(
+                    transport::protocol::RequestToChild::SendMessage {
+                        uri: msg.peer_uri,
+                        payload: Opaque::new(),
+                    },
+                );
+                self.network_gateway.publish(cmd)?;
```

Comment:
Shouldn't this be a request instead of a publish, so we can respond with a ConnectResult with the matching request_id from core?